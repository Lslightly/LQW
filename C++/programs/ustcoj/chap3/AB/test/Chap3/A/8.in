CCCAbBBaaCabcCbCaaCBabAaCBaAabbCBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAcCbAbbBBaBB
C??????A#B??BaCAB???Ac
