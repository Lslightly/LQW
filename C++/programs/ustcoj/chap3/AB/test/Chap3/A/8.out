0 CCCAbBBaaCabcCbCaaCBabAaCBaAabbCBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAc
1 CCAbBBaaCabcCbCaaCBabAaCBaAabbCBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAc
9 CabcCbCaaCBabAaCBaAabbCBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAc
13 CbCaaCBabAaCBaAabbCBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAc
15 CaaCBabAaCBaAabbCBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAc
31 CBbcBBaaBABCBaaAaCAAaaCcbbBbABaCABcCcAc
42 CBaaAaCAAaaCcbbBbABaCABcCcAc
